src/Utilities/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,55 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>PKV004</DiagnosticId>
+    <Target>.NETCoreApp,Version=v2.0</Target>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>PKV004</DiagnosticId>
+    <Target>.NETFramework,Version=v4.6.1</Target>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>PKV004</DiagnosticId>
+    <Target>.NETFramework,Version=v4.6.2</Target>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>PKV004</DiagnosticId>
+    <Target>.NETFramework,Version=v4.6.3</Target>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>PKV004</DiagnosticId>
+    <Target>.NETStandard,Version=v2.0</Target>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>PKV004</DiagnosticId>
+    <Target>Tizen,Version=v4.0</Target>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>PKV004</DiagnosticId>
+    <Target>UAP,Version=v10.0.15064</Target>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>PKV004</DiagnosticId>
+    <Target>Xamarin.PlayStation3,Version=v0.0</Target>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>PKV004</DiagnosticId>
+    <Target>Xamarin.PlayStation4,Version=v0.0</Target>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>PKV004</DiagnosticId>
+    <Target>Xamarin.PlayStationVita,Version=v0.0</Target>
```

Comment:
What do these mean, and why are they necessary? Is it an artifact of moving from net6 to net7 between 17.3 and 17.4? If so should we bump the baseline to 17.4?

Reply:
Yes good catch! We should update the baseline to 17.4 to get cleaner baseline files.

Reply:
This is an indication that the package supports `netstandard2.0` as a contract but doesn't offer compatible runtime assemblies. It lists the set of platforms that the packages doesn't at run time but at compile time.

<img width="194" alt="image" src="https://user-images.githubusercontent.com/7412651/207821985-9d39ae83-a936-4d52-b5a4-ff0dd819567f.png">

In dotnet/runtime we would call this a packaging bug as offering a contract without an implementation results in a poor experience at consumption time. Developers would be able to compile to but fail during runtime with an Exception. In dotnet/runtime we intentionally offer PNSE assemblies that throw a PlatformNotSupportedExceptions with a localized message.

Reply:
Ah, great. I'll push a commit with some comments explaining our reasoning behind this decision.

Reply:
cc @ericstj

Reply:
We stopped using ref in runtime packages because there were too many scenarios that didn't work well with it.  There are old design time tools that try to use reflection (which loads for execution by default) to load references - or actually try to run them.  

It's also not a great user experience to have a ref with no lib since it would mean that the only indication someone got of missing support is a runtime FileNotFoundException at an unpredictable point when the JIT decides to load the assembly.

The pattern we follow in runtime is to include a PNSE implementation assembly in lib/netstandard2.0 that can be used for reference.  We use SupportedOSPlatform attributes to help message when things won't work on other platforms.

Aside: the number of errors we emit here is larger than it needs to be.  We could just emit a single error about the most specific TFM missing (netstandard2.0) and then could provide the other TFMs in the error message.  This might make the error less noisy for the customer and thus more likely to be understood.

Reply:
> The pattern we follow in runtime is to include a PNSE implementation assembly in lib/netstandard2.0 that can be used for reference.

I think it would make sense for us to do this for all TFs, since to use MSBuild APIs you must run within the context of an installed VS or SDK, so the "normal library" versions of everything are useless. However, that's a build and packaging problem I don't think we're willing to tackle at the moment.

Reply:
> Aside: the number of errors we emit here is larger than it needs to be. We could just emit a single error about the most specific TFM missing (netstandard2.0) and then could provide the other TFMs in the error message. This might make the error less noisy for the customer and thus more likely to be understood.

I do think that would have helped me.